app/components/RegionDropdown/RegionDropdown.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect } from "react";
+import { useStickyState } from "../../hooks/useStickyState";
+import styles from "./RegionDropdown.module.scss";
+
+export type DhbRegionId = "auckland" | "northIsland" | "southIsland";
+type Option = {
+  id: DhbRegionId;
+  label: string;
+};
+
+const options: Option[] = [
+  { id: "auckland", label: "Auckland DHBs" },
+  { id: "northIsland", label: "North Island DHBs" },
+  { id: "southIsland", label: "South Island DHBs" },
+];
+
+type RegionDropdownProps = {
+  onChange: (id: DhbRegionId) => void;
+};
+
+export const RegionDropdown = ({ onChange }: RegionDropdownProps) => {
+  const [region, setRegion] = useStickyState<DhbRegionId>(
+    "regionPreference",
+    "auckland"
+  );
+
+  const handleChange = (id: DhbRegionId) => {
+    setRegion(id);
+    onChange(id);
+  };
+
+  useEffect(() => {
+    onChange(region);
+  }, []);
```

Comment:
Is there a particular reason to sync state rather than lift state up a level?

Reply:
Mainly because we're persisting state in `local-storage` and I'm delaying the render until its mounted on client side.

The idea was the parent will wait for this "initial state" before attempting to show/filter the data. If we lift up then would have to delay access to initial state somewhere.

Def open to better solutions

Reply:
Ah gotcha - what's the implication for server rendering? We can't show anything till on the client?

Reply:
Pretty much 😬 

Or we forego the persistence for now and default Auckland regardless to keep it simple for now

Reply:
🤔 Yeh interesting problem - I think removing persistence might be a good move. Created it as a diff ticket so we could iterate on it